### ISSUE TITLE
System - Privacy Consent conflict with K2 system

### ISSUE THREAD

**Issue by cenobitz** [getk2/k2#509]
Hello.
New installation, upgraded to k2 latest build beta, latest Joomla. When enabling system-privacy consent and opening website it is redirecting me to profile page to accept or not Privacy Policy. The problem is that when K2 system is enabled there is no way to save option. The page is reloading and nothing changes. When You try to open different website url it is still redirecting. So I made few test and changed templates to default, disabled other plugins... what I noticed that no matter what template plugin is enabled you must disable K2 system to make SYSTEM PRIVACY CONSENT working. So i found few posts on web with the same problem and asking You here - are You going to fix this issue?
Regards, Maciej

**Comment by cenobitz** [comment]
The problem is when the K2 system option > site > enable frontend users is set to YES. Than privacy consent is not working. Please fix it.

**Comment by fevangelou** [comment]
That is correct. And it's on my to-do list to adapt for upcoming K2 releases.

**Comment by PhilETaylor** [comment]
28th Jan 2021, and this issue still seems to be a problem,

Linking https://www.joomlaworks.net/forum/k2-en/49841-joomla-3-9-0-k2-system-plugin-does-not-let-user-save-accept-privacy-policy which was 2 years 2 months ago... 

Today I was unable to get this working for a client, even with @cenobitz settings so will  set the option "Enable K2 user profile" to "No" in K2's component settings as @fevangelou advises 11 months ago in that thread.

**Comment by PhilETaylor** [comment]
I might also point out that JCE doesn't load right on the K2 User profile page, and 2FA options are also missing on the K2 User Profile page :-(
